downloads/release-notes/1.2.7/RELEASE_NOTE.md

```diff
@@ -22,9 +22,92 @@ However, you need to use the following commands instead of the above if you have
 
 If you have not installed jBallerina, then download the [installers](https://ballerina.io/downloads/) to install.
 
+## Language
```

Comment:
If we don't have Language side fixes shall we remove this topic?

Reply:
Done.